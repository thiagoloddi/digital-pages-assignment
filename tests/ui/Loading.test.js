import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../app/components/ui/Loading';

it('renders correctly showing', () => {
  const tree = renderer
    .create(<Loading show={true} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly hidden', () => {
  const tree = renderer
    .create(<Loading show={false} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});