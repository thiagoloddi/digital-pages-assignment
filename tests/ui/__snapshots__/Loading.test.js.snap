// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly hidden 1`] = `
<div
  className="Loading-loading-0-1-1 Loading-loading-0-1-4"
>
  <img
    className="Loading-gif-0-1-2"
    src="/assets/progress.gif"
  />
</div>
`;

exports[`renders correctly showing 1`] = `
<div
  className="Loading-loading-0-1-1 Loading-loading-0-1-3"
>
  <img
    className="Loading-gif-0-1-2"
    src="/assets/progress.gif"
  />
</div>
`;
